src/domain/Chat/Chat.module.scss

```
@@ -86,55 +113,61 @@
-      transition: all 0.2s ease;
-
-      &:hover {
-        box-shadow: 0 6px 16px rgba(f.color(orange-700), 0.4);
-        transform: translateY(-1px);
-      }
     }
 
-    &__message--bot {
+    &__message--other {
+      @extend %message-bubble;
       background: rgba(255, 255, 255, 0.8);
```

Stylelint (17.2.0)

[error] Expected "0.8" to be "80%" (alpha-value-notation)

(alpha-value-notation)